reg/udral/README.md

```diff
@@ -236,6 +236,34 @@ It is not necessary to include the leading and trailing name component separator
 Registers that are not known to the auto-configuration authority are to be silently ignored,
 which permits addition of implementation-defined ports without breaking compatibility with standard services.
 
+
+#### Type signature register
+
+In many use cases it is desirable to have the ability to statically check the compatibility between sender and 
```

Comment:
A bit of a nit but also a question, "statically check" generally means compile-time. We're adding a runtime mechanism here. Should this be "dynamically check" or just "check" or am I not understanding the proposal?

Reply:
It is indeed a runtime check that is intended to happen once (as opposed to on every message transmission) during system initialization (or even configuration). 
Do you have any wording suggestion that would improve on "statically" and still express this distinction?

Reply:
>"statically check" generally means compile-time

I agree with Vadim here -- I am reading "static" as a "check that is performed before the system is operational", so it is not a runtime check, strictly speaking.